### DISCUSSION TITLE
How to make setup/exe file based on windows Universal build

### DISCUSSION THREAD

**Discussion by Barumugam3** [2fast-team/2fast#60] [Q&A]
Hi,

I cloned project, build and debugged working fine. But I am trying to export or create installer package getting failed lot of dependency error coming. Unable to create installer package. Please share step to proceed further thanks.

**Comment by Barumugam3** [comment]
Team checked similar old issue reported and mentioned to use installer package. But inside package certificate have one year valid. It is possible to increase more than a year?

**Reply by jp-weber** [reply]
Which way do you tried to create a installer?
The normal way for an UWP app is with a msix bundle:
https://learn.microsoft.com/en-us/windows/msix/package/packaging-uwp-apps
